src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -647,19 +650,19 @@ private void getCanvasResponse(HttpServletRequest request,
 		BufferedReader rd = new BufferedReader(new InputStreamReader(canvasResponse.getEntity().getContent()));
 
 		String linkValue = null;
-		
+
 		Header[] headers = canvasResponse.getAllHeaders();
 		for (Header header : headers) {
 			M_log.debug("Key : " + header.getName() 
-			      + " ,Value : " + header.getValue());
+					+ " ,Value : " + header.getValue());
```

Comment:
Is this how Eclipse formats the lines?  If not someone will come along and accidentally reformat them generating a false change.